### DISCUSSION TITLE
Missing kata solutions

### DISCUSSION THREAD

**Discussion by lostman** [codewars/codewars.com#2994] [General]
Hi,

I solved some kata, and they appear in my profile under "Completed," but when I click on the link, and then browse "Solutions, and then select VIEW "Mine," all I can see is "No solutions."

Is this a bug, or is there a reason it happens?

An example would be the "Light switch."

https://www.codewars.com/kata/63306fdffa185d004a987b8e

FWIW, if I click on "Train again," I can see my solution under "Past Solutions." It's confusing to see it one place and not the other.

An explanation would help!

Thanks 🙏

**Comment by Kacarott** [comment]
Sometimes this happens due to a bug. If you contact a mod (easiest way is on Discord) then they can make the solution visible again. What is your username on Codewars?

Edit: I found your account, have a look if you can see the solutions now.

**Reply by lostman** [reply]
I can see them. Thanks, @Kacarott.

**Reply by iRobot42** [reply]
Hey, fellow warriors!
I have the same problem, until I train kata again or refactor after submit.
@lostman, how did you handle this problem?

**Reply by lostman** [reply]
@iRobot42 I didn't. @Kacarott fixed this for some of my solutions, but the problem repeats for newer solutions that I submit 😢.

**Reply by iRobot42** [reply]
Thanks for your answer!
I found the root of the problem: if kata is solved but not yet submitted, just don't check the other solutions (already unlocked).
Happy coding! 👊
